input/definitions/StructureDefinition/StructureDefinition-cqf-recorded.json

```diff
@@ -0,0 +1,86 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "cqf-recorded",
+  "text": {
+    "status": "extensions"
+  },
+  "url": "http://hl7.org/fhir/StructureDefinition/cqf-recorded",
+  "version": "5.0.0",
+  "name": "CQFNotDoneRecorded",
+  "title": "CQF Not Done Recorded",
+  "status": "active",
+  "experimental": false,
+  "date": "2018-08-22",
+  "publisher" : "HL7 International / FHIR Infrastructure",
+  "contact": [
+    {
+      "name" : "Clinical Quality Information WG",
+      "telecom": [
+        {
+          "system": "url",
+          "value" : "http://www.hl7.org/Special/committees/fiwg"
+        }
+      ]
+    }
+  ],
+  "description": "Captures the recorded date of the event.",
+  "jurisdiction": [
+    {
+      "coding": [
+        {
+          "system": "urn:iso:std:iso:3166",
+          "code": "US"
+        }
+      ]
+    }
+  ],
+  "fhirVersion": "4.0.1",
+  "mapping": [
+    {
+      "identity": "rim",
+      "uri": "http://hl7.org/v3",
+      "name": "RIM Mapping"
+    }
+  ],
+  "kind": "complex-type",
+  "abstract": false,
+  "context": [
+    {
+      "type": "element",
+      "expression": "Resource"
```

Comment:
This should be limited to only those resources that we expect this is usable on. In particular, it should not be usable on any resource that already has a "recorded date" element. In QICore, we only use this on Communication, MedicationAdministration, MedicationDispense, and Procedure.